examples/LoRaWANABP/LoRaWANABP.ino

```diff
@@ -0,0 +1,91 @@
+/*
+  LoRaABP.ino
+
+  Establish a connection on a LoRaWAN network in ABP mode.
+  Exchange data to/from the network.
+
+*/
+
+#include "LoRaWANNode.h"
+
+#define FRAME_DELAY 300000  // in ms. Every 5 minutes by default.
+
+HardwareSerial SerialLora(PA_1, PA_0);
+
+// Device address, network & application keys
+const char devAddr[] = "ef00cb01";
+const char nwkSKey[] = "abcdef0123456789abcdef0123456789";
+const char appSKey[] = "0123456789abcdef0123456789abcdef";
+
+// Data send
+char frameTx[] = "Hello world!";
+
+void setup()
+{
+  Serial.begin(9600);
+  Serial.println("-- LoRaWAN ABP sketch --");
+
+  // Enable the USI module and set the radio band.
+  while(!node.begin(&SerialLora, LORA_BAND_EU_868)) {
```

Comment:
as commented in previous commit, maybe lora rather than node would make it more clear?